src/initialize.cpp

```diff
@@ -262,6 +262,10 @@ int parse_command_line(int argc, char* argv[])
         print_version();
         return OPENMC_E_UNASSIGNED;
 
+      } else if (arg == "-b" || arg == "--build-info") {
+        print_build_info();
```

Comment:
Rather than having a separate flag here, what would you think about printing this info as part of `-v` / `--version`. This would be similar to, e.g. `gcc -v`, which shows version and build info.

Reply:
I quite like this idea, the printed info does contain some version info for things like the compiler version so that feels like it fits. It would also save a little bit of code